plugins/ltac/rewrite.mli

```diff
@@ -28,13 +28,16 @@ type unary_strategy =
   | Bottomup | Topdown | Progress | Try | Any | Repeat
 
 type binary_strategy =
-  | Compose | Choice
+  | Compose
```

Comment:
Since there is only one `binary_strategy`, is there any point keeping it around?